easybuild/easyconfigs/b/BerkeleyGW/BerkeleyGW-1.2.0-intel-2018a.eb

```diff
@@ -0,0 +1,45 @@
+easyblock = 'ConfigureMake'
+
+name = 'BerkeleyGW'
+version = '1.2.0'
+
+homepage = 'http://www.berkeleygw.org'
+description = """The BerkeleyGW Package is a set of computer codes that calculates the quasiparticle
+ properties and the optical responses of a large variety of materials from bulk periodic crystals to
+ nanostructures such as slabs, wires and molecules."""
+
+toolchain = {'name': 'intel', 'version': '2018a'}
+toolchainopts = {'usempi': True, 'openmp': True}
+
+source_urls = ['http://www.berkeleygw.org/releases/']  # requires registration
+sources = ['BGW-%(version)s.tar.gz']
+checksums = [
+    '1305dc8587af666fe437bc2561a9106b2b0bcdbe91980b8f1ae7bbd491ce1e25',  # BGW-1.2.0.tar.gz
+    '7bd3cc229693a1bd5fe6547f55046bee68c8c50f29f8ca6945f443b04ce2ca38',  # BerkeleyGW-1.2.0_fix_intent.patch
+]
+
+patches = ['BerkeleyGW-1.2.0_fix_intent.patch']
```

Comment:
@migueldiascosta Use `%(version)s` here, and move this line above `checksums` (you didn't use `--inject-checksums`?)

Reply:
I did use `--inject-checksums` when I added the patch, but the source checksum was already there from earlier